psq/datastore_storage_test.py

```diff
@@ -0,0 +1,88 @@
+# Copyright 2015 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from gcloud import datastore
+from mock import Mock
+from psq.datastore_storage import DatastoreStorage
+from psq.task import Task
+
+
+def test_ops():
+    datastore_mock = Mock()
+    storage = DatastoreStorage(datastore_mock)
+
+    t = Task('1', sum, (), {})
+
+    # shouldn't store queued, retrying, or started tasks.
+    storage.put_task(t)
+    assert not datastore_mock.put.called
+
```

Comment:
re: naming, unclear exactly why datastore shouldn't put this one. maybe if it was named `t_queued` or something, it would be a bit clearer.